### PR TITLE
Removes most Ruins from Spawning (Runtime Fix)

### PR DESCRIPTION
**Pull request by WickedCybs**
They cause unbeatable issues due to walls deleting themselves.

### CODE REVIEW

File: code/modules/maps/planet_types/barren.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		/datum/map_template/ruin/exoplanet/crashed_sol_shuttle_01,
 		/datum/map_template/ruin/exoplanet/crashed_skrell_shuttle_01,
 		/datum/map_template/ruin/exoplanet/mystery_ship_1,
-		/datum/map_template/ruin/exoplanet/crashed_satelite,
+		/datum/map_template/ruin/exoplanet/crashed_satellite ,
```
Comment (SleepyGemmy) [inline code/modules/maps/planet_types/barren.dm:21]:
```suggestion
		/datum/map_template/ruin/exoplanet/crashed_satellite,
```

File: maps/random_ruins/exoplanets/asteroid/asteroid.dm [hunk 2]
```diff
@@ -81,20 +81,20 @@
...
 	spawn_cost = 2
 	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
-	suffixes = list("asteroid/mystery_ship/mystery_ship_unique")
+	suffixes = list("asteroid/mystery_ship/mystery_ship_unique.dmm")
 
 	ruin_tags = RUIN_HUMAN|RUIN_VOID
 
-/datum/map_template/ruin/exoplanet/crashed_satelite
-	name = "Crashed Satelite"
+/datum/map_template/ruin/exoplanet/crashed_satellite
```
Comment (SleepyGemmy) [inline maps/random_ruins/exoplanets/asteroid/asteroid.dm:88]:
```suggestion
/datum/map_template/ruin/exoplanet/crashed_satellite
```

File: maps/random_ruins/exoplanets/asteroid/asteroid.dm [hunk 3]
```diff
@@ -81,20 +81,20 @@
...
 	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
-	suffixes = list("asteroid/mystery_ship/mystery_ship_unique")
+	suffixes = list("asteroid/mystery_ship/mystery_ship_unique.dmm")
 
 	ruin_tags = RUIN_HUMAN|RUIN_VOID
 
-/datum/map_template/ruin/exoplanet/crashed_satelite
-	name = "Crashed Satelite"
+/datum/map_template/ruin/exoplanet/crashed_satellite 
+	name = "Crashed satellite "
```
Comment (SleepyGemmy) [inline maps/random_ruins/exoplanets/asteroid/asteroid.dm:89]:
```suggestion
	name = "Crashed Satellite"
```

File: maps/random_ruins/exoplanets/asteroid/asteroid.dm [hunk 4]
```diff
@@ -81,20 +81,20 @@
...
 	id = "crashed_satelite"
-	description = "A crashed satelite."
+	description = "A crashed satellite ."
```
Comment (SleepyGemmy) [inline maps/random_ruins/exoplanets/asteroid/asteroid.dm:91]:
```suggestion
	description = "A crashed satellite."
```

File: maps/random_ruins/exoplanets/asteroid/asteroid.dm [hunk 5]
```diff
@@ -81,20 +81,20 @@
...
 	template_flags = TEMPLATE_FLAG_CLEAR_CONTENTS|TEMPLATE_FLAG_NO_RUINS
 
 	spawn_weight = 1
 	spawn_cost = 1
 	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_NEW_ANKARA, SECTOR_AEMAQ, SECTOR_SRANDMARR, SECTOR_NRRAHRAHUL, SECTOR_GAKAL, SECTOR_UUEOAESA)
-	suffixes = list("asteroid/satelite_crash/satelite_crash_unique")
+	suffixes = list("asteroid/satellite _crash/satellite_crash_unique.dmm")
```
Comment (SleepyGemmy) [inline maps/random_ruins/exoplanets/asteroid/asteroid.dm:97]:
```suggestion
	suffixes = list("asteroid/satellite_crash/satellite_crash_unique.dmm")
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
!review

**Pull request closed without merging**